tests/api/metrics/test_app.py

```diff
@@ -0,0 +1,25 @@
+def test_unsupported_events(client):
```

Comment:
as those tests are not necessarily related to metrics, this particular file can be moved to something around the lines of `tests/api/app/test_app.py`

Reply:
Done